bench/wtperf/wtperf.h

```diff
@@ -73,12 +77,15 @@ typedef struct {
 	uint64_t truncate_pct;		/* Truncate Percent */
 	uint64_t truncate_count;	/* Truncate Count */
 	int64_t update_delta;		/* Value size change on update */
+	int64_t modify_delta;		/* Value size change on modify */
+	bool	modify_force_update; /* Do force update instead of modify */
 
 #define	WORKER_INSERT		1	/* Insert */
 #define	WORKER_INSERT_RMW	2	/* Insert with read-modify-write */
 #define	WORKER_READ		3	/* Read */
 #define	WORKER_TRUNCATE		4	/* Truncate */
 #define	WORKER_UPDATE		5	/* Update */
+#define	WORKER_MODIFY		6   /* Modify */
```

Comment:
Alphabetize. And fix the spacing for the comment.

Reply:
corrected.